BackofficeBundle/Resources/views/layout.html.twig

```diff
@@ -84,6 +84,6 @@
             pageSetUp();
         });
     </script>
-
+    {{ orchestra_tinymce_init() }}
```

Comment:
I think (not sure) that the only instance of tinymce_init() is located in form_div_layout.html.twig, so it would be better to update the twig extension rather than creating a new one.